spec/replicate_spec.cr

```diff
@@ -9,6 +9,7 @@ DIST_REP2_VOLDATA_JSON  = File.read("samples/dist_replica2.json")
 REP3_VOLDATA_JSON       = File.read("samples/replica3.json")
 DIST_REP3_VOLDATA_JSON  = File.read("samples/dist_replica3.json")
 TIEBREAKER_VOLDATA_JSON = File.read("samples/tiebreaker.json")
+ARBITER_VOLDATA_JSON    = File.read("samples/storage_unit_arbiter.json")
```

Comment:
Use different sample here. Copy replica3 json and add arbiter count

Reply:
Done